source/renderer/app/components/news/NewsFeed.js

```diff
@@ -157,7 +162,7 @@ export default class NewsFeed extends Component<Props, State> {
             <SVGInline svg={closeCrossThin} />
           </button>
         </div>
-        <div className={styles.newsFeedContainer}>
+        <div className={newsFeedContainerStyles}>
```

Comment:
@DeeJayElly please use `newsFeedContainerClasses` instead of `newsFeedContainerStyles ` here 🙏

Reply:
Fixed